src/deque.py

```diff
@@ -0,0 +1,38 @@
+"""Deque Composition."""
+from dll import DoubleLinkedList
+
+
+class Deque(object):
+    """Class deque for deque composition."""
+
+    def __init__(self):
+        """Init for deque."""
+        self._new_dll = DoubleLinkedList()
+
+    def append(self, val):
+        """Enqueue function for queue pushes value to head."""
+        return self._new_dll.push(val)
+
+    def appendleft(self, val):
+        """Append vals to the front of deque."""
+        return self._new_dll.append(val)
+
+    def pop(self):
+        """Pop from dll."""
+        return self._new_dll.pop()
+
+    def popleft(self):
+        """Shift for popleft."""
+        return self._new_dll.shift()
+
+    def peek(self):  # pragma no cover
+        """Return a new value without dequeuing it."""
+        print(self._new_dll.head.data)
```

Comment:
You shouldn't need to print here. A simple return will do. It should also be tested and covered by your tests. Same for peekleft